manifests/a/Argotronic/ArgusMonitor/7.0.8.2762/Argotronic.ArgusMonitor.installer.yaml

```
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2024/Jun/05
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 7.0.8.2762
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- ProductCode: ArgusMonitor
+  DisplayVersion: 7.0.8.2760

```

```suggestion
  DisplayVersion: 7.0.8.2762
```
Purely guessing here, please let me know if incorrect.